### PR TITLE
Setup sample project for local testing

### LINKED ISSUE #12: Local deployment - add sample project setup
See https://github.com/FELToken/federated-learning-token/blob/main/scripts/deploy.py for original sample project setup.

### PR DESCRIPTION
**Pull request by Breta01**
Resolve #12

### CODE REVIEW

File: scripts/project.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""Module for deploying project contract."""
+from brownie import FELToken, ProjectContract, ProjectManager, accounts, config
+
+from scripts.crypto import decrypt_nacl, encrypt_nacl, export_public_key, get_current_secret
+
+
+def deploy_project(owner):
+    """Deploy project contract."""
+    token = FELToken[-1]
```
Comment (FilipMasar) [inline scripts/project.py:9]: if you want to use latest contract shouldn't it be `FELToken[0]` ?

Comment (FilipMasar) [reply scripts/project.py:9]: Maybe it is irrelevant, but in `map.json` https://eth-brownie.readthedocs.io/en/stable/deploy.html#the-deployment-map 
`The list for each contract is sorted by the block number of the deployment with the most recent deployment first.`

File: scripts/project.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    # Using dummy test secret
+    secret = b"Initial secret must be 32 bytes."
+    assert len(secret) == 32
+
+    public_key = export_public_key(owner.private_key[2:])
+    ciphertext = encrypt_nacl(public_key, secret)
+    project = ProjectContract.deploy(token, public_key, list(ciphertext), {"from": owner})
+
+    manager = ProjectManager[-1]
```
Comment (FilipMasar) [inline scripts/project.py:19]: same here

File: scripts/project.py (hunk 1 above)
Comment (Breta01) [reply scripts/project.py:9]: But here (https://github.com/eth-brownie/brownie/issues/1003#issuecomment-802880435) they write that the contract object `FELToken` is list where `-1` index is the latest deployment. Here they also write that deployments are appended to the list:
https://eth-brownie.readthedocs.io/en/stable/core-contracts.html#deploying-contracts
There is no explicit word about correct indexing in documentation, so I am not sure.

**Pull request merged**